### PR TITLE
Add a note on health checkpoint API for redash

### PR DESCRIPTION
**Pull request by kurianbenoy-aot**
Fix for mentioning docs for health checkpoint API: https://github.com/getredash/redash/issues/5040

### CODE REVIEW

File: src/pages/kb/open-source/dev-guide/docker.md [hunk 1]
```diff
@@ -61,6 +61,12 @@ docker-compose run --rm server create_db
 docker-compose run --rm postgres psql -h postgres -U postgres -c "create database tests"
 ```
 
+### Health Check for Installation
+
+{% callout warning %}
+After your installation is complete, you can do the healthcheck by calling **/ping** API endpoint.
```
Comment (susodapop) [inline src/pages/kb/open-source/dev-guide/docker.md:67]:
```suggestion
After your installation is complete, you can do the healthcheck by calling `/ping` API endpoint.
```

File: src/pages/kb/open-source/dev-guide/setup.md [hunk 2]
```diff
@@ -92,6 +92,12 @@ This will result in a Flask web server listening on port `5000`, Webpack
 rebuilding changes to the frontend, RQ worker ready to run queries and RQ
 scheduler to enqueue periodic tasks.
 
+## Health Check for Installation
+
+{% callout warning %}
+After your installation is complete, you can do the healthcheck by calling **/ping** API endpoint.
```
Comment (susodapop) [inline src/pages/kb/open-source/dev-guide/setup.md:98]:
```suggestion
After your installation is complete, you can do the healthcheck by calling `/ping` API endpoint.
```

File: src/pages/kb/open-source/setup.md [hunk 3]
```diff
@@ -172,6 +172,12 @@ Redash uses environment variables for configuration. For a full list of environm
 If this is a production setup, you should enforce HTTPS and make sure you set the cookie secret (see [instructions]({% link _kb/open-source/admin-guide/https-ssl-setup.md %})).
 {% endcallout %}
 
+## Health Check for Installation
+
+{% callout warning %}
+After your installation is complete, you can do the healthcheck by calling **/ping** API endpoint.
```
Comment (susodapop) [inline src/pages/kb/open-source/setup.md:178]:
```suggestion
After your installation is complete, you can do the healthcheck by calling `/ping` API endpoint.
```

**Review by susodapop — CHANGES_REQUESTED** [review]
Looks good. Just style nits.

**Comment by kurianbenoy-aot** [comment]
Oh that was quick @susodapop. Have addressed your style comments.

**Review by susodapop — APPROVED** [review]

**Comment by kurianbenoy-aot** [comment]
First PR merged to Redash. Super happy about it :)

**Pull request merged**